platform/src/components/icons/mod.rs

```diff
@@ -0,0 +1,76 @@
+use dioxus::prelude::*;
+
+#[component]
+pub fn Search(width: String, height: String, color: String) -> Element {
+    rsx! {
+        svg {
+            "viewBox": "0 0 21 20",
```

Comment:
```suggestion
            view_box: "0 0 21 20",
```
Rust 컨벤션을 최대한 활용하면 좋을듯 합니다.

Reply:
수정했습니다!